### ISSUE TITLE
PAI doesn't clean job container

### ISSUE THREAD

**Issue by mzmssg** [Microsoft/pai#2354]
**Organization Name**: 

Micorsoft

**Short summary about the issue/question**:

PAI doesn't clean job contaienrs.

**OpenPAI Environment**:
- OpenPAI version: master

**Anything else we need to know**:
Root cause: 
In #1108, we remove `--rm` as launching job container, so docker daemon won't clean containers. Confirmed with @wangdian that cleaner won't clean these containers. Then there is no container clean mechanism now. 
Residual container examples:
```
42af191d4a5c        openpai/pai.example.tensorflow                        "/bin/bash /pai/boot…"   About an hour ago   Exited (0) About an hour ago                                                    admin-test-sleep-container_e44_1552954684228_0103_01_000002
4fe903811bca        openpai/pai.example.tensorflow                        "/bin/bash /pai/boot…"   About an hour ago   Exited (143) About an hour ago                                                  admin-test-kill-job3-container_e44_1552954684228_0100_01_000002

```

**Comment by mzmssg** [comment]
It's a blocking issue

**Comment by fanyangCS** [comment]
Why removing -rm?

**Comment by mzmssg** [comment]
@fanyangCS 
This pr need `docker inspect` to get container OOM status.

**Comment by fanyangCS** [comment]
Ok. After the inspection, we could rm it. Right? Please also check whether the debug mode is affected.

**Comment by mzmssg** [comment]
@fanyangCS 
Nope, all commands in yarn container might be interrupted, so any clean action in the yarn container is unreliable. 
We need dockerd(with `--rm`)  or cleaner(implement by ourselves) to manage the residual containers out of yarn container.

**Comment by fanyangCS** [comment]
https://github.com/Microsoft/pai/pull/2355

**Comment by mzmssg** [comment]
Fixed in #2355
